src/libsyntax/parse/mod.rs

```diff
@@ -33,7 +33,6 @@ pub mod attr;
 
 pub mod common;
 pub mod classify;
-pub mod obsolete;
```

Comment:
We may not want to remove this module entirely, it's a useful pattern which we can continue to move obsolete parsing to over time. The contents of the module can certainly be trimmed down, however.